.changeset/sour-ligers-deny.md

```diff
@@ -0,0 +1,5 @@
+---
+"@mysten/sui.js": minor
+---
+
+Pure args can be passed directly
```

Comment:
```suggestion
"@mysten/sui.js": patch
---

Allow passing Pure args directly in Move call
```

We are following the semver semantics for Alpha and beta (0.x.y) : https://github.com/dbrock/semver-howto#alpha-and-beta-0xy, incrementing y for every backward-compatible change, and x for every incompatible change. So this should be a patch change since it's backward compatible